trio/_tests/test_exports.py

```diff
@@ -422,19 +444,19 @@ def lookup_symbol(symbol):
 
         # I have not researched why these are missing, should maybe create an issue
         # upstream with jedi
-        if tool == "jedi" and sys.version_info >= (3, 11):
-            if class_ in (
-                trio.DTLSChannel,
-                trio.MemoryReceiveChannel,
-                trio.MemorySendChannel,
-                trio.SSLListener,
-                trio.SocketListener,
-            ):
-                missing.remove("__aenter__")
-                missing.remove("__aexit__")
-            if class_ in (trio.DTLSChannel, trio.MemoryReceiveChannel):
-                missing.remove("__aiter__")
-                missing.remove("__anext__")
+        # if tool == "jedi" and sys.version_info >= (3, 11):
+        # if class_ in (
+        #    trio.DTLSChannel,
+        #    trio.MemoryReceiveChannel,
+        #    trio.MemorySendChannel,
+        #    trio.SSLListener,
+        #    trio.SocketListener,
+        # ):
+        #    missing.remove("__aenter__")
+        #    missing.remove("__aexit__")
+        # if class_ in (trio.DTLSChannel, trio.MemoryReceiveChannel):
+        #    missing.remove("__aiter__")
+        #    missing.remove("__anext__")
```

Comment:
the 3.11&3.12 failures are because I accidentally pushed this - I think my local environment updated `jedi` or something.

Will look at the windows fails